CocktailRecipes/CocktailRecipes/Features/Home/Views/SmallDrinkCell.swift

```diff
@@ -9,57 +9,79 @@
+        label.font = UIFont.preferredFont(forTextStyle: .body)
+        label.contentHuggingPriority(for: .vertical)
+        return label
+    }()
 
-	required init?(coder: NSCoder) {
-		fatalError("init(coder:) has not been implemented")
-	}
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        setupView()
+    }
 
-	override func prepareForReuse() {
-		super.prepareForReuse()
-		imageView.image = nil
-		label.text = nil
-	}
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
 
-	private func setupView() {
-		contentView.addSubview(imageView)
-		contentView.addSubview(label)
+    override func prepareForReuse() {
+        super.prepareForReuse()
+        imageView.image = nil
+        rankLabel.text = nil
+        titleLabel.text = nil
+    }
 
-		NSLayoutConstraint.activate([
-			imageView.leadingAnchor.constraint(equalTo: contentView.leadingAnchor, constant: 16),
-			imageView.topAnchor.constraint(equalTo: contentView.topAnchor, constant: 8),
-			imageView.bottomAnchor.constraint(equalTo: contentView.bottomAnchor, constant: -8),
-			imageView.widthAnchor.constraint(equalTo: imageView.heightAnchor)
-		])
+    private func setupView() {
+        contentView.addSubview(imageView)
+        contentView.addSubview(rankLabel)
+        contentView.addSubview(titleLabel)
 
-		NSLayoutConstraint.activate([
-			label.leadingAnchor.constraint(equalTo: imageView.trailingAnchor, constant: 24),
-			label.trailingAnchor.constraint(equalTo: contentView.trailingAnchor, constant: -16),
-			label.centerYAnchor.constraint(equalTo: contentView.centerYAnchor)
-		])
+        NSLayoutConstraint.activate([
+            imageView.leadingAnchor.constraint(equalTo: contentView.leadingAnchor, constant: 16),
+            imageView.topAnchor.constraint(equalTo: contentView.topAnchor, constant: 8),
+            imageView.bottomAnchor.constraint(equalTo: contentView.bottomAnchor, constant: -8),
+            imageView.widthAnchor.constraint(equalTo: imageView.heightAnchor)
+        ])
+
+        NSLayoutConstraint.activate([
+            rankLabel.leadingAnchor.constraint(equalTo: imageView.trailingAnchor, constant: 24),
+            rankLabel.topAnchor.constraint(equalTo: contentView.topAnchor, constant: 10)
+        ])
+
+        NSLayoutConstraint.activate([
+            titleLabel.leadingAnchor.constraint(equalTo: imageView.trailingAnchor, constant: 24),
+            titleLabel.trailingAnchor.constraint(equalTo: contentView.trailingAnchor, constant: -16),
+            titleLabel.centerYAnchor.constraint(equalTo: contentView.centerYAnchor)
+        ])
 	}
 
-	func configure(image: UIImage?, text: String) {
+    func configure(image: UIImage?, text: String, rank: Int?) {
 		imageView.image = image
-		label.text = text
+        titleLabel.text = text
+
+        if let rank = rank {
+            rankLabel.text = "\(rank)"
+            rankLabel.isHidden = false
```

Comment:
I think we can remove this and the else and combine it to one line `rankLabel.isHidden = rank == nil`

Reply:
done